executor/join.go

```diff
@@ -198,8 +199,8 @@ func (e *HashJoinExec) fetchOuterRows() {
 		}
 
 		select {
-		case <-e.ctx.GoCtx().Done():
-			return
+		// case <-e.ctx.GoCtx().Done():
```

Comment:
why comment out these lines ?

Reply:
Because `ctx.GoCtx()` method is removed, and we can't get a context to cancel the calculation.
As I've said

> 2. some executor maybe not cancelable, because the break of context chain.

It will be fix in next PR soon, if `Next()` changed to `Next(goctx.Context)`.
Comment the code rather than remove them, I can find it more easy next time.

Reply:
Because `ctx.GoCtx()` method is removed, and we can't get a context to cancel the calculation.
As I've said

> 2. some executor maybe not cancelable, because the break of context chain.

It will be fix in next PR soon, if `Next()` changed to `Next(goctx.Context)`.
Comment the code rather than remove them, I can find it more easy next time.